alpha/action/migrations/000_bundle_object_to_csv_metadata.go

```diff
@@ -0,0 +1,50 @@
+package migrations
+
+import (
+	"encoding/json"
+
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+
+	"github.com/operator-framework/operator-registry/alpha/declcfg"
+	"github.com/operator-framework/operator-registry/alpha/property"
+)
+
+var bundleObjectToCSVMetadata = newMigration(
+	"bundle-object-to-csv-metadata",
+	"migrates bundles' `olm.bundle.object` to `olm.csv.metadata`",
+	func(cfg *declcfg.DeclarativeConfig) error {
```

Comment:
Nit: If we wanted the names and descriptions of all the migrations in one place, we could do something like the following instead.

Here: get rid of the `newMigration` wrapping and directly declare the migration function:
```go
func migrateBundleObjectsToCSVMetadata(cfg *declcfg.DeclarativeConfig) error { ... }
```

In `migrations.go` in line 74 where we setup `allMigrations`, add the `newMigration` wrapping:
```go
newMigration("bundle-object-to-csv-metadata", `migrates bundles' "olm.bundle.object" to "olm.csv.metadata"`, migrateBundleObjectsToCSVMetadata)
```

Reply:
LOL!  I /almost/ did that already, but I was making a concerted effort to not make 'nice to have' changes in my update, in order to get this in sooner.  
But now I think there's no excuse not to make the changes. 😁